docs/proposals/pipeline/pipeline.md

```diff
@@ -368,35 +368,55 @@ For more details, see [Tekton signed-provenance-tutorial](https://tekton.dev/doc
 
 #### Kurator Pipeline cli
 
-##### 1. Pipeline Commands
+We are focused on providing the viewing logs of Pipeline executions. Therefore, we have concentrated the CLI's features on this core requirement.
 
-- **`pipeline list`**
-  
-    Display all current Pipelines.
+For viewing information about Pipeline objects, users can directly utilize existing tools like kubectl.
```

Comment:
We can say this is not the goal of this proposal.

Reply:
because user can just use `kubectl get Pipelines` to get config about pipeline.
we offer something different. 

For example, we can list pipeline executions for a specified namespace for the user, then the user can select one and get aggregated logs from multiple tasks-run pods.

These are features not available in kubectl. In fact, these are basic functionalities in Tekton's own CLI, but we do not want users to have to learn and use an additional CLI specifically for Tekton